### DISCUSSION TITLE
Change Start Button

### DISCUSSION THREAD

**Discussion by vidalv** [Open-Shell/Open-Shell-Menu#1538] [General]
I try to replace the starter button from Windows 11 with that of Open-Shell. When I replace it with mine, both (the original + My_one) are superimposed! 
How to make the original disappear? 
Thanks.

**Comment by Slate66** [comment]
open shell custom start buttons don't replace the win11 start button it overlaps it instead so in your case I think it's a matter of getting it to fit and hide the win 11 start button correctly. For example, for some custom ones to fit correctly you have to change the custom buttons size because that also shifts its place

**Comment by vidalv** [comment]
Hello and thank you @Slate66 for the information, this is probably a new Open-Shell feature! 
I did not have this on Windows 10. In fact, I am able to get both the menu, Windows 11 and one selected by the open-shell configuration, and it is not bad afterwards! I keep this like it. 
Thanks for help.

**Reply by Slate66** [reply]
your welcome!
